### PR TITLE
Automate Communities bundle creation

### PR DESCRIPTION
**Pull request by clobrano**
- Add Makefile recipe to build RH community bundle
- Add Makefile recipe to build K8S community bundle
- Automate RH/OKD and K8s community bundle release with Github Action

### CODE REVIEW

**Comment by clobrano** [comment]
/hold 
wait for https://github.com/medik8s/.github/pull/4

File: Makefile [hunk 1]
```diff
@@ -268,6 +271,10 @@ verify-previous-version: ## Verifies that PREVIOUS_VERSION variable is set
 bundle-reset-date: ## Reset bundle's createdAt
 	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ${BUNDLE_CSV}
 
+.PHONY: bundle-community-rh
+bundle-community-rh: bundle-community ## Generate bundle manifests and metadata customized to Red Hat community release
+	echo -e "\n  # Annotations for OCP\n  com.redhat.openshift.versions: \"v${OCP_VERSION}\"" >> bundle/metadata/annotations.yaml
+
```
Comment (razo7) [inline Makefile:277]: Run `$(MAKE) bundle-update` at the end as in `bundle-community` target?

Comment (clobrano) [reply Makefile:277]: Why do it twice?

Comment (razo7) [reply Makefile:277]: You are right. I haven't noticed that `bundle-community-rh` already calls `bundle-community`...

**Comment by clobrano** [comment]
I think we don't need to wait for CI tests, as this change is not tested by any of them, we could just merge it

**Pull request merged**